doc/cephadm/services/monitoring.rst

```diff
@@ -598,7 +598,7 @@ webhook urls like so:
     service_type: alertmanager
     spec:
       user_data:
-        default_webhook_urls:
+        webhook_urls:
```

Comment:
The below line refers to `default_webhook_urls` but the example doesn't.. is that correct? I'm little bit confused on whether we are changing the name of the field or not.